invenio_rdm_records/fixtures/data/vocabularies.yaml

```diff
@@ -49,3 +49,7 @@ funders:
 awards:
   pid-type: awa
   data-file: vocabularies/awards.yaml
+# code meta controlled vocabularies
+developmentstatus:
```

Comment:
If we go for contrib I would say yes, but (IIRC) the `FixturesEngine` does not support looking into folders that are not `/app_data` or `/data` (root under fixtures)

Reply:
Also not a big fan of this as a top-level term. I don't know how much flexibility we have with generic vocabulary names, but could we do something like:

- `code:developmentStatus`, i.e. mapping directly to the custom field name(space)
- `codemeta/developmentStatus`

Path-wise it could be `vocabularies/contrib/development_status.yaml` or similar (and would still work based on the 2 prefixes we have).

Reply:
Sounds much better, thanks for the suggestion!